lib/procutil/signal.go

```diff
@@ -11,6 +11,6 @@ import (
 // Returns the caught signal.
 func WaitForSigterm() os.Signal {
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, os.Interrupt, syscall.SIGTERM)
+	signal.Notify(ch, os.Interrupt, syscall.SIGTERM, syscall.SIGHUP)
```

Comment:
please notice that WaitForSigterm is used in other project for graceful shutdown without handling the type of the signal, so this change will introduce graceful shutdown by syscall.SIGHUP

Reply:
ok, got it